### PR TITLE
add topic to implicitTransferLog

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: proto/types/action.proto [hunk 1]
```diff
@@ -308,9 +308,14 @@ message BlockEvmTransfer {
 // 2. Withdraw bucket from nsv2 protocol
 message ImplicitTransferLog {
   message Transaction {
-    string amount = 1;
-    string sender = 2;
-    string recipient = 3;
+    enum TxType {
```
Comment (yutongp) [inline proto/types/action.proto]: maybe `ActionType`

File: proto/types/action.proto [hunk 2]
```diff
@@ -308,9 +308,14 @@ message BlockEvmTransfer {
...
+      EVM_TRANSFER = 0;
```
Comment (yutongp) [inline proto/types/action.proto]: maybe `EXECUTION` and `STAKE_WITHDRAW` reflect to their action name

File: proto/types/action.proto (hunk 1 above)
Comment (dustinxie) [reply proto/types/action.proto]: ok

File: proto/types/action.proto (hunk 2 above)
Comment (dustinxie) [reply proto/types/action.proto]: changed

File: proto/types/action.proto [hunk 3]
```diff
@@ -308,9 +308,10 @@ message BlockEvmTransfer {
 // 2. Withdraw bucket from nsv2 protocol
 message ImplicitTransferLog {
   message Transaction {
-    string amount = 1;
-    string sender = 2;
-    string recipient = 3;
+    bytes topic = 1;
```
Comment (yutongp) [inline proto/types/action.proto:311]: Not sure if it need repeated or not

**Review by yutongp — APPROVED** [review]

**Pull request merged**